Python/pywarpx/picmi.py

```diff
@@ -545,6 +545,12 @@ def initialize_inputs(self):
             pywarpx.warpx.do_electrostatic = 'labframe'
             pywarpx.warpx.self_fields_required_precision = self.required_precision
             pywarpx.warpx.self_fields_max_iters = self.maximum_iterations
+            pywarpx.geometry.potential_lo = [
```

Comment:
Thanks for getting the update to the picmi interface started for this change. More changes will be needed, for example this needs to be generalized to work in 2D and 3D.

Also, the potential_xmin etc would be something good to add to the picmi standard. @RemiLehe do you agree? I would add it to the various Grid classes.

Reply:
I've added a `Geometry.py` file to instantiate `boundary = Bucket('boundary')`, which was needed in order to change the boundary condition specification to `boundary.potential_lo` and `boundary.potential_hi`.